flight-icons/README.md

```diff
@@ -0,0 +1,35 @@
+# flight-icons
+
+An addon for the icons from `flight`. Currently in development. This addon can be used in React or Ember apps.
+
+## Installation
+
+```bash
+yarn add @hashicorp/flight-icons
+```
+
+## Usage
+
+We intend for icons to be used at a 16 or 24 `viewBox` size.
+
+### How to choose 16 or 24 `viewBox` size
```

Comment:
@dizzyup @didoo took a stab at this from the meeting, what do you think of this wording as a first stab? also anyone is open to reply, but thought you were the most relevant

currently we have no readme for https://www.npmjs.com/package/@hashicorp/flight-icons, so was thinking some initial guidance would be better than nothing, so that we can empower external teams to test the addons on their own